configurations/pipeline_configuration.py

```diff
@@ -42,6 +42,16 @@
             )
         )
     ],
+    csv_sources=[
+        CSVSource(
+            "gs://avf-project-datasets/2022/RVI-ELECTIONS/recovered_hormuud_2022_03_12_to_2022_03_18_de_identified.csv",
```

Comment:
Is there any reason for not incorporating the data after 18th yet? The logs you shared suggest there are a large number of messages from the 19th, 20th, and 21st which could already be integrated

Reply:
I'll incorporate it in another PR, I was focusing on e01 in this PR.